examples/tv-casting-app/darwin/MatterTvCastingBridge/MatterTvCastingBridge.xcodeproj/project.pbxproj

```diff
@@ -423,11 +423,7 @@
 					/* Compile time codegen would need this: 
 					   "$(TEMP_DIR)/out/gen/src/controller/data_model/zapgen/", 
 					 */
-					"$(CHIP_ROOT)/zzz_generated/",
 					"$(CHIP_ROOT)/zzz_generated/app-common",
-					"$(CHIP_ROOT)/zzz_generated/controller-clusters",
-					/* darwin-specific bypassing compile time codegen for header inclusion */
-					"$(CHIP_ROOT)/zzz_generated/darwin/controller-clusters",
```

Comment:
If you're not using `zzz_generated/darwin` anymore, please remove it.  This was the only consumer.

Reply:
Or I can do it in a followup.  Also needs the line generating it removed from `scripts/tools/zap_regen_all.py`.  Just let me know.

Reply:
Added this commit: https://github.com/project-chip/connectedhomeip/pull/26762/commits/9b6ce72842263f988f021e9624ad99972a04bfc6